app/views/mentors/van_drivers.html.erb

```diff
@@ -1,6 +1,29 @@
-<%= javascript_include_tag 'driver_form.js' %>
+<h1>Drivers for <%= @term.is_a?(Quarter) ? @term.title : @term.to_param %></h1>
 
-<h1>Current Trained Drivers</h1>
+<table id="current_drivers">
+	<thead>
+		<tr>
+			<th>Name</th>
+			<th>E-mail</th>
+			<th>Training Date</th>
+			<%- if Customer.link_to_uw? %><th>UWFS Training Date</th><% end -%>
+            <th class="centered">Signed Form?</th>
+            <th class="centered">Convictions?</th>
+			<th colspan=4>Functions</th>
+		</tr>
+	</thead>
+
+	<tbody>
+		<%= render :partial => "mentor_driver", :collection => @current_drivers %>
+		<%- if @mentors.empty? -%>
+			<tr class="empty">
+				<td colspan=10>No <%= Customer.mentors_label %> are currently trained to drive.</td>
+			</tr>
+		<% end -%>
+	</tbody>
+</table>
+
+<h1>All Trained Drivers</h1>
 
 <table>
```

Comment:
I'm going to see if the staff wants this table here, if they do I'll factor out the table into a partial, otherwise I'll delete this table.